src/biogeochem/CNBalanceCheckMod.F90

```diff
@@ -345,10 +374,10 @@ subroutine NBalanceCheck(this, bounds, num_soilc, filter_soilc, &
          write(iulog,*)'net flux                 = ',(col_ninputs(c)-col_noutputs(c))*dt
          write(iulog,*)'inputs,ffix,nfix,ndep    = ',ffix_to_sminn(c)*dt,nfix_to_sminn(c)*dt,ndep_to_sminn(c)*dt
          write(iulog,*)'outputs,ffix,nfix,ndep   = ',smin_no3_leached(c)*dt, smin_no3_runoff(c)*dt,f_n2o_nit(c)*dt
-        
+      
          
          
-         call endrun(msg=errMsg(sourcefile, __LINE__))
+         call endrun(msg=errMsg(sourcefile, __LINE__))  !zgdu
```

Comment:
Similarly please remove the `!zgdu` comment here.

Reply:
done.